### ISSUE TITLE
easy way to plot loss for autoencoder model

### ISSUE THREAD

**Issue by jramak** [yzhao062/pyod#47]
Is there an easy way to plot the loss as a function of epochs for the pyod autoencoder module? I want to visualize this to decide how many epochs to train for.

**Comment by yzhao062** [comment]
This might be hard since a customized function which would not fall into the umbrella of all detectors. I would say you may want to check clf.history_ once it is fitted. This is a standard keras history output. See definition below.

` self.history_ = self.model_.fit(X_norm, X_norm, epochs=self.epochs,
                                        batch_size=self.batch_size,
                                        shuffle=True,
                                        validation_split=self.validation_size,
                                        verbose=self.verbose).history
`

**Comment by jramak** [comment]
Great, thank you
